Formula/o/openai-whisper.rb

```diff
@@ -79,24 +81,22 @@ class OpenaiWhisper < Formula
   end
 
   resource "urllib3" do
-    url "https://files.pythonhosted.org/packages/43/6d/fa469ae21497ddc8bc93e5877702dca7cb8f911e337aca7452b5724f1bb6/urllib3-2.2.2.tar.gz"
-    sha256 "dd505485549a7a552833da5e6063639d0d177c04f23bc3864e41e5dc5f612168"
+    url "https://files.pythonhosted.org/packages/ed/63/22ba4ebfe7430b76388e7cd448d5478814d3032121827c12a2cc287e2260/urllib3-2.2.3.tar.gz"
+    sha256 "e7d814a81dad81e6caf2ec9fdedb284ecc9c73076b62654547cc64ccdcae26e9"
   end
 
   def install
-    python3 = "python3.12"
-    venv = virtualenv_create(libexec, python3)
+    ENV["LLVM_CONFIG"] = Formula["llvm@16"].opt_bin/"llvm-config"
+    venv = virtualenv_install_with_resources without: "numba"
 
     # We depend on pytorch, but that's a separate formula, so install a `.pth` file to link them.
-    # This needs to happen _before_ we try to install torchvision.
-    site_packages = Language::Python.site_packages(python3)
-    pytorch = Formula["pytorch"].opt_libexec
-    (venv.site_packages/"homebrew-pytorch.pth").write pytorch/site_packages
+    # NOTE: This is an exception to our usual policy as building `pytorch` is complicated
+    site_packages = venv.site_packages.relative_path_from(venv.root)
```

Comment:
Any reason not to use `Language::Python.site_packages` here?

Reply:
Mainly to avoid hard coding the `python3.12` reference.

`virtualenv_install_with_resources` has Python detection logic internally, so that should provide a reliable way of selecting correct Python. 

If we hard code `python3.12` reference, then it should really be used in all DSL, e.g.
```rb
python3 = "python3.12"
virtualenv_install_with_resources using: python3, ...
```

However, that is usually unnecessary and adds extra maintenance to formula.

Reply:
You could do
```ruby
Language::Python.site_packages(venv.root/"bin/python3")
```